pkg/storaged/drive/smart-details.jsx

```diff
@@ -82,31 +80,30 @@ const SmartActions = ({ smart_info }) => {
         smart_info.SmartSelftestAbort({});
     };
 
-    const testDisabled = !superuser.allowed || smartSelftestStatus === "inprogress";
+    const testDisabled = smartSelftestStatus === "inprogress";
 
-    return (
-        <Dropdown
-                toggle={toggleRef => <MenuToggle ref={toggleRef} icon={<EllipsisVIcon />} variant="plain" aria-label={_("SMART actions")} onClick={() => setKebabOpen(!isKebabOpen)} isExpanded={isKebabOpen} />}
-                isPlain
-                isOpen={isKebabOpen}
-                id="smart-actions"
-        >
-            <DropdownItem key="smart-short-test"
-                        isDisabled={testDisabled}
-                        onClick={() => { setKebabOpen(false); runSelfTest('short') }}>
+    const actionItems = (
+        <DropdownList>
+            <StorageMenuItem excuse={testDisabled}
```

Comment:
The `excuse` is supposed to be a string that is shown in a tooltip, but here it is a boolean. Does that work? Or do we get a broken tooltip?